inc/vcf/report_writer.hpp

```diff
@@ -35,18 +35,32 @@ namespace ebi
             virtual void write_warning(Error &error) = 0;
     };
 
-    class StdoutReportWriter : public ReportWriter
+    class FileReportWriter : public ReportWriter
     {
         public:
+            FileReportWriter(std::string filename)
+            {
+                fout.open(filename, std::ios::out);
+            }
+
+            ~FileReportWriter()
+            {
+                fout.close();
```

Comment:
although the ofstream can be opened in the constructor and is closed when it's destroyed, I prefer having those explicitly stated like they are now. 

However, don't you think `file` would be a better name than `fout`?